nestkernel/nest_time.h

```diff
@@ -537,6 +562,9 @@ class Time
   {
     return ld_round( ms * Range::STEPS_PER_MS );
   }
+
+private:
+  unsigned long steps_;
```

Comment:
And C is memory usage for the extra members? I wouldn't expect to see anything here -- the issue wouldn't be memory usage, but the extra demand on bandwidth between RAM and caches, and multiple registers used unnecessarily, plus confusing the optimizer by operating on multiple operands that are glued together.